site/source/cookbook/command-line/create-new-app.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+category: command-line
+title: Creating a new Dojo 2 application
+overview: Create a new Dojo 2 application from scratch
+---
+
+# Creating a new Dojo 2 application
+
+## Objective
+
+This recipe demonstrates the process of creating a new Dojo 2 application.
+
+## Procedure
```

Comment:
I did feel there are more words than necessary here, for example, with a recipe, all i want to do is get to the crux of the issue. In this case, I want minimal information to achieve my goal:

Something like: 

---
1. Install the Dojo CLI if you have not already with the command: `npm install @dojo/cli -g`
2. Create a new app with the command: `dojo create --name your-application-name`
3. Navigate to your newly created Dojo 2 application directory
4. Build the application with the command `dojo build --watch`
5. View your application in the browser at `http://localhost:9999`
---

Maybe?

I feel that users who need extra guidance, e.g. to be told a command is entered in the "terminal or command-line" or for those who need an explanation + example that "{app-name}" should be substituted with _their_ application name, or that the command to use for running tests (which is an unrelated task and could be a recipe in itself) is `dojo test`...I think tutorials are perfect for that and provide much more guidance/hand-holding.

Reply:
Or maybe include the `dojo test` bit in the additional resources at the bottom?